### DISCUSSION TITLE
Occasionally getting "TypeError: Load failed"

### DISCUSSION THREAD

**Discussion by jfacoustic** [Shopify/storefront-api-feedback#237] [Help]
Occasionally when I hit the Shopify Storefront API, I see "TypeError: Load failed" in my Sentry logs.   This means that Shopify either sent a CORS error or never sent a response, although I've been unable to reproduce the bug locally, so I can't get more details.  Any ideas what's going on?  This happens fairly infrequently, but enough for my team to be concerned about losing leads. 

I'm using NextJS/client-side api calls.

**Comment by jfacoustic** [comment]
Anybody?

**Comment by allenjiang17** [comment]
Did anyone find a solution/get more info on this? Also getting Load failed issues, somewhat infrequently, but this is happening to us as well
